docs/user-guides/using-tables.md

```diff
@@ -122,11 +122,11 @@ For more information, please refer to [Self-optimizing quota](../self-optimizing
 ### Adjust optimizing parameters
 
 You can manually set parameters such as execution interval, task size, and execution timeout for different types of Optimize. 
-For example, to set the execution interval for Self-optimize of the major type, you can do the following:
+For example, to set the execution interval for Self-optimize of the minor type, you can do the following:
```

Comment:
This sentence is quite ChatGPT type.

I think 'to set the execution interval for minor optimizing type' would be better, and it could be more friendly if we add a link to minor optimizing docs here.

Reply:
> This sentence is quite ChatGPT type.
> 
> I think 'to set the execution interval for minor optimizing type' would be better, and it could be more friendly if we add a link to minor optimizing docs here.

Yes, it is better if we do that. In addition, I don't find any info about 'self-optimizing.major.trigger.interval'. Maybe ams doesn't support this config, right?

Reply:
Yes, 'self-optimizing.major.trigger.interval' is a deprecated config.